projects/packages/blaze/src/class-dashboard-config-data.php

```diff
@@ -8,6 +8,7 @@
 namespace Automattic\Jetpack\Blaze;
 
 use Automattic\Jetpack\Connection\Manager;
+use Automattic\Jetpack\Constants;
```

Comment:
Since we're using a new package here, could you add it as a dependency to the Blaze package?
https://github.com/Automattic/jetpack/blob/a771ddd7d937bdb7bd490d687d42aa939df922d6/projects/packages/blaze/composer.json#L8

Right now it's not an issue because the Jetpack plugin requires that `automattic/jetpack-constants` package already, but in the future the Blaze package may be used on sites that do not use that package, so we'll want to account for that.

After you've updated the list of dependencies, you'll want to run `tools/fixup-project-versions.sh` to ensure the lock files are updated too.

Reply:
Thanks @jeherve! Hope I got you right, 38b199cfeb781f6c339b7c1b5fc605a99e8c5e0d contains your suggestion.